lms/templates/components/tabbed/tabbed_view.underscore

```diff
@@ -1,5 +1,5 @@
 <div class="page-content">
     <nav class="page-content-nav" role="tablist"></nav>
```

Comment:
just seeing this now, we do not want a role=tablist here.  that complicates things tremendously and requires that we add all of our own JS behavior.  As long as we handle focus properly, which we are now doing, we do not need to make this a tablist.